source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -7,15 +7,31 @@ weight: 70
 
 ## CSS
 
-### Import all the CSS
+### Add the CSS file to your HTML
+
+If you decide to add the CSS to your HTML, first either:
```

Comment:
are the bullet points an either/or option or steps you have to do one after the other?

Reply:
Can do either of them

I copied this pattern from the JavaScript section further down which is nearly identical

Happy to improve both?

>### Add the JavaScript file to your HTML
>
>If you decide to add the JavaScript to your HTML, first either:

Reply:
OK. How about - 
If you decide to add the CSS to your HTML, you can choose to either:
Another option - 
If you decide to add the CSS to your HTML, you can do one of the following:
I'm basically just removing the 'first' so it doesn't sound like the start of a list you have to do in a specific order.